### DISCUSSION TITLE
Welcome to tailpress Discussions!

### DISCUSSION THREAD

**Discussion by jeffreyvr** [tailpress/tailpress#22] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by tdecoy** [comment]
Hey Jeff

Thanks so much for this repo. It's exactly what I needed, as the existing ones are outdated and unnecessarily cumbersome.

**Reply by jeffreyvr** [reply]
Glad you like it @tdecoy! :)
